opentabletdriver/PKGBUILD

```diff
@@ -6,32 +6,38 @@
 
 pkgname=opentabletdriver
 _pkgname=OpenTabletDriver
-_lpkgname=opentabletdriver
-_spkgname=otd
 pkgver=0.6.4.0
-pkgrel=7
+pkgrel=8
 pkgdesc="A cross-platform open source tablet driver"
 arch=('x86_64')
 url="https://opentabletdriver.net"
-license=('LGPL3')
+
+# license FIXME tracked upstream:
+# https://github.com/OpenTabletDriver/OpenTabletDriver/issues/3571
+license=('LGPL3') # FIXME: use SPDX format
```

Comment:
Please put your SPDX Format

Reply:
The proper identifier hasn't been decided yet by the OTD team. I believe that the next point release will be released some time next week, so we can drop this by then. There doesn't seem to be any updates other than some cleanups in this sync so I'll just close this.